### PR TITLE
Products lactose-free

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -20578,6 +20578,11 @@ hu:Füstölt sajtok
 nl:Gerookte kazen, Rookkaas
 wikidata:en:Q2632885
 
+<en:Cheeses
+en:Lactose-free cheeses
+fr:Fromages sans lactose
+es:Quesos sin lactosa
+
```
Comment (VaiTon) [inline taxonomies/categories.txt]:
```suggestion
it:Formaggi senza lattosio

```

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -21181,6 +21186,15 @@ country:en:Greece
 el:Xynomyzithra Kritis
 country:en:Greece
 
+<en:Cheeses
+en:Halloumi
+de:Halloumi
+es:Halloumi
+fr:Halloumi
+sv:Halloumi
```
Comment (VaiTon) [inline taxonomies/categories.txt]:
```suggestion
sv:Halloumi
it:Halloumi
```

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -23793,6 +23808,11 @@ lad:Yaúrt
 pnns_group_2:en:Milk and yogurt
 wikidata:en:Q13317
 
+<en:Yogurts
+en:Lactose-free yogurts
+es:Yogures sin lactosa
+fr:Yaourts sans lactose
+
```
Comment (VaiTon) [inline taxonomies/categories.txt]:
```suggestion
it:Yogurt senza lattosio

```

**Comment by VaiTon** [comment]
@stephanegigandet @aleene any thoughts?

File: taxonomies/categories.txt [hunk 4]
```diff
@@ -21181,6 +21186,15 @@ country:en:Greece
 el:Xynomyzithra Kritis
 country:en:Greece
 
+<en:Cheeses
+en:Halloumi
+de:Halloumi
+es:Halloumi
+fr:Halloumi
```
Comment (aleene) [inline taxonomies/categories.txt]: nl:Halloumi

File: taxonomies/categories.txt [hunk 5]
```diff
@@ -20578,6 +20578,11 @@ hu:Füstölt sajtok
 nl:Gerookte kazen, Rookkaas
 wikidata:en:Q2632885
 
+<en:Cheeses
+en:Lactose-free cheeses
+fr:Fromages sans lactose
+es:Quesos sin lactosa
```
Comment (aleene) [inline taxonomies/categories.txt]: nl:Lactose-vrije kaas, Lactose-vrije kazen

File: taxonomies/categories.txt [hunk 6]
```diff
@@ -23793,6 +23808,11 @@ lad:Yaúrt
 pnns_group_2:en:Milk and yogurt
 wikidata:en:Q13317
 
+<en:Yogurts
+en:Lactose-free yogurts
+es:Yogures sin lactosa
+fr:Yaourts sans lactose
```
Comment (aleene) [inline taxonomies/categories.txt]: nl:Lactose-vrije yoghurt, Lactose-vrije yoghurts, Yoghurt zonder lactose, Yoghurts zonder lactose

File: taxonomies/categories.txt (hunk 5 above)
Comment (VaiTon) [reply taxonomies/categories.txt]:
```suggestion
es:Quesos sin lactosa
nl:Lactose-vrije kaas, Lactose-vrije kazen
```

File: taxonomies/categories.txt (hunk 4 above)
Comment (VaiTon) [reply taxonomies/categories.txt]:
```suggestion
fr:Halloumi
nl:Halloumi
```

File: taxonomies/categories.txt (hunk 6 above)
Comment (VaiTon) [reply taxonomies/categories.txt]:
```suggestion
fr:Yaourts sans lactose
nl:Lactose-vrije yoghurt, Lactose-vrije yoghurts, Yoghurt zonder lactose, Yoghurts zonder lactose
```

**Pull request merged**